libvast/src/system/archive.cpp

```diff
@@ -147,6 +147,11 @@ archive(archive_type::stateful_pointer<archive_state> self, path dir,
             namespace defs = defaults::system;
             self->delayed_send(self, defs::telemetry_rate,
                                telemetry_atom::value);
+          },
+          [=](erase_atom, ids erase) {
```

Comment:
`s/erase/xs`? The `erase_atom` already defines the function. A second `erase` seems redudant.